api-admin/src/main/java/bio/terra/pearl/api/admin/controller/StudyEnvironmentController.java

```diff
@@ -68,17 +68,47 @@ public ResponseEntity<Object> getKitTypes(
       String portalShortcode, String studyShortcode, String envName) {
     AdminUser adminUser = requestService.requireAdminUser(request);
     EnvironmentName environmentName = EnvironmentName.valueOfCaseInsensitive(envName);
-    List<KitType> kitTypes =
-        studyEnvExtService.getKitTypes(adminUser, portalShortcode, studyShortcode, environmentName);
+    PortalStudyEnvAuthContext authContext =
+        PortalStudyEnvAuthContext.of(adminUser, portalShortcode, studyShortcode, environmentName);
+
+    List<KitType> kitTypes = studyEnvExtService.getKitTypes(authContext);
+    return ResponseEntity.ok(kitTypes);
+  }
+
+  @Override
+  public ResponseEntity<Object> patchKitTypes(
```

Comment:
I don't think we're in the habit of using `patch` in method names.  I'd just as soon call this "updateKitTypes", but I could be persuaded otherwise